Cargo.toml

```diff
@@ -24,9 +24,9 @@ bitflags = ">=1.0.1"
 libc = ">=0.2.39"
 
 vmm-sys-util = ">=0.3.1"
-vm-memory = "0.6"
+vm-memory = ">=0.6"
```

Comment:
`vm-memory = "0.6"` should allow to upgrade to any version " >= 0.6 && < 1.0". Why does it cause multi-versions of vm-memory being used?

Reply:
Does "cargo update" help to resolve the multi-version dependent crates issue?